bindings/wasm/CHANGELOG.md

```diff
@@ -19,6 +19,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
-## 0.1.0 - 2023-MM-DD
+## 1.0.0-rc.0 - 2023-MM-DD
```

Comment:
date wasn't changed

Reply:
Ah yeah sorry, changed it in the wrong readme, Python should stay without exact date